### PR TITLE
add Nominal APR and Net APR

### PR DESCRIPTION
**Pull request by 20xharley**
Add Chart Nominal APR and Net APR.
- Nominal APR Daily = Fee Received / LLP Value * 100
- Net APR Daily = (Fee Received + PnL vs Trader) / LLP Value * 100

Update table llp history

### CODE REVIEW

File: apps/llp-tracker-ui/src/pages/LpView/TableLLPHistory.tsx [hunk 1]
```diff
@@ -298,4 +351,18 @@ const CurrencyView = ({ value, options }: { value: number; options?: Partial<For
   );
 };
 
+const PercentView = ({ value, options }: { value: number; options?: Partial<FormatOption> }) => {
+  return (
+    <span className={value == 0 ? '' : value > 0 ? 'positive' : 'negative'}>
+      {value == 0 ? '' : value > 0 ? '+' : ''}
```
Comment (ukin-level) [inline apps/llp-tracker-ui/src/pages/LpView/TableLLPHistory.tsx]: You can use: {value > 0 ? '+' : ''}

File: apps/llp-tracker-ui/src/pages/LpView/index.tsx [hunk 2]
```diff
@@ -137,15 +138,17 @@ const LpView: React.FC = () => {
                     <div className="relative">
                       <strong className="pb-8px block text-16px">{t.name}</strong>
                       <div className="text-14px">
-                        {lpBalances && lpPrices ? (
+                        {isLoadingPrice || isLoadingPrice ? (
```
Comment (ukin-level) [inline apps/llp-tracker-ui/src/pages/LpView/index.tsx]: Duplicate check isLoadingPrice state

**Review by xBillCipher — APPROVED** [review]
LGTM!

**Review by 0xhari-s — APPROVED** [review]
seem good

**Pull request merged**